src/app/App.tsx

```diff
@@ -30,12 +37,15 @@ function App() {
 
           <nav>
             <ul>
-              <li>
+              {/* <li>
                 <Link to="/">All documents</Link>
-              </li>
+              </li> */}
```

Comment:
Coding style: commented discarded code can be removed during PR.

Reply:
removed